File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x)
     {
        if(x<0) return false;
        
        int rev=0,temp=x;
       while(x>0)
       {
        rev=(rev*10)+x%10;
        x/=10;
       }
       return rev==temp;
       
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        n=x
        s=0
        while(x>0):
            s=s*10+(x%10)
            x=x//10
        if(n==s):
            return True
        else:
            return False

        